azure-pipelines.yml

```diff
@@ -76,9 +76,13 @@ stages:
             PYTHON_VERSION: '3.6'
             NPY_USE_BLAS_ILP64: '1'
             USE_OPENBLAS: '1'
-          Accelerate:
-            PYTHON_VERSION: '3.6'
-            USE_OPENBLAS: '0'
+          # Disable this job: the azure images do not create the problematic
+          # symlink from Accelerate to OpenBLAS. We still have the test
+          # at import to detect a buggy Accelerate, just cannot easily trigger
+          # it with azure.
+          # Accelerate:
```

Comment:
Do we need to keep this commented code ?

Reply:
I was not sure how the change would be received, so preferred to comment it out rather than remove for easy reversal.

Reply:
I agree keeping this commented, for now makes sense. If we do not see any issues from broader community, we can uncomment it.